### PR TITLE
Spring version upgrade changes

### PR DESCRIPTION
**Pull request by veswantht-wtt-egov**
maven upgrade to bpa-services following the document
https://digit-discuss.atlassian.net/wiki/spaces/EPE/pages/487096370/Maven+version+upgrade

### CODE REVIEW

File: bpa-calculator/pom.xml [hunk 1]
```diff
@@ -3,14 +3,13 @@
 	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 	<parent>
-		<groupId>org.springframework.boot</groupId>
-		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>1.5.22.RELEASE</version>
-		<relativePath /> <!-- lookup parent from repository -->
+	  <groupId>org.springframework.boot</groupId>
```
Comment (tarunlalwani) [inline bpa-calculator/pom.xml:6]: Missing `CHANGELOG.md`, See https://github.com/egovernments/core-services/blob/5f9cd5d5af9b5d02f80302ac219226d44d792f5b/egov-idgen/CHANGELOG.md

File: bpa-calculator/pom.xml [hunk 2]
```diff
@@ -3,14 +3,13 @@
...
+	  <artifactId>spring-boot-starter-parent</artifactId>
+	  <version>2.2.6.RELEASE</version>
 	</parent>
 	<groupId>egov.bpa</groupId>
 	<artifactId>bpa-calculator</artifactId>
-	<version>0.0.1-SNAPSHOT</version>
+	<version>1.1.1-SNAPSHOT</version>
```
Comment (tarunlalwani) [inline bpa-calculator/pom.xml:12]: Should be `1.1.0`

File: bpa-services/src/main/resources/application.properties [hunk 3]
```diff
@@ -34,6 +36,7 @@ spring.kafka.consumer.key-deserializer=org.apache.kafka.common.serialization.Str
 spring.kafka.consumer.group-id=egov-bpa-services
 spring.kafka.producer.key-serializer=org.apache.kafka.common.serialization.StringSerializer
 spring.kafka.producer.value-serializer=org.springframework.kafka.support.serializer.JsonSerializer
+spring.kafka.listener.missing-topics-fatal=false
```
Comment (tarunlalwani) [inline bpa-services/src/main/resources/application.properties:39]: Add `management.endpoints.web.base-path=/` also for health check backward compatibility

File: bpa-services/src/main/resources/application.properties [hunk 4]
```diff
@@ -16,15 +18,15 @@ spring.datasource.platform=postgresql
...
-flyway.baseline-on-migrate=true
-flyway.outOfOrder=true
-flyway.locations=db/migration/main
-flyway.enabled=true
+spring.flyway.user=postgres
+spring.flyway.password=empower
+spring.flyway.table=public
+spring.flyway.baseline-on-migrate=true
+spring.flyway.outOfOrder=true
+spring.flyway.locations=classpath:db/migration/main
```
Comment (tarunlalwani) [inline bpa-services/src/main/resources/application.properties:28]: Value should be `classpath:/db/migration/main`

File: bpa-calculator/pom.xml (hunk 1 above)
Comment (phanikiran-wtt-egov) [reply bpa-calculator/pom.xml:6]: added change log for bpa-services and bpa-calculator with 

# 1.0.0
* Base version

File: bpa-services/src/main/resources/application.properties (hunk 3 above)
Comment (phanikiran-wtt-egov) [reply bpa-services/src/main/resources/application.properties:39]: management.endpoints.web.base-path=/
management.endpoints.web.path-mapping.health=healthcheck

adding above two

**Pull request merged**